src/controller/AutoCommissioner.cpp

```diff
@@ -663,35 +663,32 @@ CHIP_ERROR AutoCommissioner::CommissioningStepFinished(CHIP_ERROR err, Commissio
             mNeedsDST = false;
             break;
         case CommissioningStage::kReadCommissioningInfo2: {
-            bool shouldReadCommissioningInfo2 =
-                mParams.GetCheckForMatchingFabric() || (mParams.GetICDRegistrationStrategy() != ICDRegistrationStrategy::kIgnore);
-            if (shouldReadCommissioningInfo2)
+
+            if (!report.Is<ReadCommissioningInfo2>())
             {
-                if (!report.Is<ReadCommissioningInfo2>())
-                {
-                    ChipLogError(
-                        Controller,
-                        "[BUG] Should read commissioning info (part 2), but report is not ReadCommissioningInfo2. THIS IS A BUG.");
-                }
+                ChipLogError(
+                    Controller,
+                    "[BUG] Should read commissioning info (part 2), but report is not ReadCommissioningInfo2. THIS IS A BUG.");
+            }
 
-                ReadCommissioningInfo2 commissioningInfo = report.Get<ReadCommissioningInfo2>();
+            ReadCommissioningInfo2 mDeviceCommissioningInfo2 = report.Get<ReadCommissioningInfo2>();
```

Comment:
This local is not a member; it should not have a name starting with "m".  Why was this renamed at all?

Reply:
Changed back